packages/atomic/create-atomic/src/fetch-page.ts

```diff
@@ -0,0 +1,39 @@
+import fetch from 'node-fetch';
```

Comment:
Use Axios, or you'll have to quell the wrath of the HTTPS Proxy (it's not fun: https://github.com/coveo/cli/pull/512)